src/Tizen.NUI.Scene3D/src/public/Controls/Model.cs

```diff
@@ -18,62 +18,65 @@
 using System;
 using System.Runtime.InteropServices;
 using System.ComponentModel;
+using Tizen.NUI;
 using Tizen.NUI.Binding;
 using Tizen.NUI.BaseComponents;
 
 namespace Tizen.NUI.Scene3D
 {
     /// <summary>
-    /// ModelView is a View to show 3D model objects.
-    /// ModelView supports to load glTF 2.0 and DLI models for the input format
+    /// Model is a Class to show 3D mesh objects.
+    /// Model supports to load glTF 2.0 and DLI models for the input format
     /// and also supports Physically Based Rendering with Image Based Lighting.
     ///
-    /// The Animations defined in the glTF or DLI models are also loaded and can be retrieved by using GetAnimation() method.
+    /// The Animations defined in the glTF or DLI are also loaded and can be retrieved by using GetAnimation() method.
     /// The number of animation is also retrieved by GetAnimationCount() method.
     ///
-    /// By default, The loaded model has it's own position and size which are defined in vertex buffer regardless of the View size.
-    /// The model can be resized and repositioned to fit to the ModelView with UseSizeOfView and UseCenterOfView properties.
+    /// By default, The loaded mesh has it's own size inferred from position of vertices.
+    /// If user set size property, the mesh will be scaled to the input size.
+    ///
+    /// Both of the default value of PivotPoint and ParentOrigin of the Model is Center.
+    ///
     /// </summary>
     /// <code>
-    /// ModelView modelView = new ModelView(modelUrl)
+    /// Model model = new Model(modelUrl)
     /// {
     ///     Size = new Size(width, height),
-    ///     PositionUsesPivotPoint = true,
-    ///     PivotPoint = PivotPoint.Center,
-    ///     ParentOrigin = ParentOrigin.Center,
-    ///     UseSizeOfView = true,
-    ///     UseCenterOfView = true,
     /// };
-    /// modelView.SetImageBasedLightSource(diffuseUrl, specularUrl, scaleFactor);
-    /// window.Add(modelView);
-    /// int animationCount = modelView.GetAnimationCount();
+    /// model.SetImageBasedLightSource(diffuseUrl, specularUrl, scaleFactor);
+    /// window.Add(model);
+    ///
+    /// int animationCount = model.GetAnimationCount();
     /// if(animationCount > 0)
     /// {
-    ///     modelView.GetAnimation(0).Play();
+    ///     model.GetAnimation(0).Play();
     /// }
     /// </code>
     // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
     [EditorBrowsable(EditorBrowsableState.Never)]
-    public class ModelView : View
+    public class Model : View
     {
-        private bool useSizeOfView = false;
-        private bool useCenterOfView = false;
+        private bool useCenterPivot = false;
```

Comment:
Are you plan to use this value? (Something like.. Whether this model use Center/Center forcely, or use it's own Anchor/ParentOrigin)

If, or not If, I think we'd better remove this unused value

Reply:
Thank you I removed it.